src/cluster/placement/storage/storage_test.go

```diff
@@ -28,6 +28,7 @@ import (
 	"github.com/m3db/m3/src/cluster/kv/mem"
 	"github.com/m3db/m3/src/cluster/placement"
 
+	"github.com/stretchr/testify/assert"
```

Comment:
nit: require instead of assert?

Reply:
What's the value of using require here? Some of these seem like tests where even if the version isn't correct we'd want to debug other failures in the test rather than have to fix a version issue and push a new commit to make further test progress

Reply:
Usually I prefer to use only one of `assert` or `require` in a test unless both are really needed